modules/conflict_resolver/test/conflict_resolverTest.php

```diff
@@ -1,48 +1,41 @@
 <?php
- /**
-  * Automated integration tests for conflict resolver module
-  *
-  * PHP Version 7
-  *
-  * @category Test
-  * @package  Loris
-  * @author   Wang Shen <wangshen.mcin@gmail.com>
-  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
-  * @link     https://github.com/aces/Loris
-  */
-use Facebook\WebDriver\WebDriverSelect;
-use Facebook\WebDriver\WebDriverBy;
- require_once __DIR__
-    . "/../../../test/integrationtests/LorisIntegrationTest.class.inc";
- /**
-  * Implements tests for conflict resolver
-  *
-  * @category Test
-  * @package  Loris
-  * @author   Wang Shen <wangshen.mcin@gmail.com>
-  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
-  * @link     https://github.com/aces/Loris
-  */
-class ConflictResolverTestIntegrationTest extends LorisIntegrationTest
+/**
```

Comment:
I'm pretty sure I didn't overwrite anything important here when rebasing, but I just want to tag you @kongtiaowang just to make sure!